docs/pages/includes/config-reference/instance-wide.yaml

```diff
@@ -0,0 +1,122 @@
+# services.
+teleport:
+    # nodename allows one to assign an alternative name this node can be
+    # reached by. By default it's equal to hostname.
+    nodename: graviton
+
+    # Data directory where Teleport daemon keeps its data.
+    # See "Storage backends" for more details
+    # (https://goteleport.com/docs/setup/reference/backends/).
+    data_dir: /var/lib/teleport
+
+    # PID file for Teleport process
+    #pid_file: /var/run/teleport.pid
+
+    # The invitation token or an absolute path to a file containing the token used
+    # to join a cluster. It is not used on subsequent starts.
+    # If using a file, it only needs to exist when teleport is first ran.
+    #
+    # File path example:
+    # auth_token: /var/lib/teleport/tokenjoin
+    #
+    # This is the same as setting join_params.method to "token", and join_params.token_name
+    # to the value of auth_token.
+    # You should only use either auth_token or join_params.
+    auth_token: xxxx-token-xxxx
+
+    # join_params are parameters to set when joining a cluster via
+    # EC2, IAM or a token.
+    #
+    # EC2 join method documentation:
+    # https://goteleport.com/docs/setup/guides/joining-nodes-aws-ec2/
+    # IAM join method documentation:
+    # https://goteleport.com/docs/setup/guides/joining-nodes-aws-iam/
+    join_params:
+        # When `method` is set to "token", it is the equivalent to using `auth_token` above.
+        # You should only use either auth_token or join_params.
+        method: "token"|"ec2"|"iam"
+
+        # If method is "iam" or "ec2", token_name will be will be the name of
+        # the joining token resource, e.g., "ec2-token" or "iam-token" as created
+        # in the Joining Nodes via EC2 or IAM guides.
+
+        # If method is "token", token_name will be the invitation token
+        # or an absolute path to a file containing the token used to join a cluster.
+        # It is not used on subsequent starts.
+        # If using a file, it only needs to exist when teleport is first ran.
+        #
+        # File path example:
+        # token_name: /var/lib/teleport/tokenjoin
+        token_name: "token-name"
+
+    # Optional CA pin of the auth server. This enables a more secure way of
+    # adding new nodes to a cluster. See "Adding Nodes to your Cluster"
+    # (https://goteleport.com/docs/setup/admin/adding-nodes).
+    ca_pin: "sha256:7e12c17c20d9cb504bbcb3f0236be3f446861f1396dcbb44425fe28ec1c108f1"
+
+    # When running in multi-homed or NATed environments Teleport Nodes need
+    # to know which IP it will be reachable at by other Nodes.
+    #
+    # This value can be specified as FQDN e.g. host.example.com
+    advertise_ip: 10.1.0.5
+
+
+    # Teleport provides HTTP endpoints for monitoring purposes. They are
+    # disabled by default but you can enable them using the diagnosis address.
+    # See the Teleport metrics reference:
+    # https://goteleport.com/docs/setup/diagnostics/metrics
```

Comment:
I just noticed while using our reference guide that this link is broken.

Reply:
```suggestion
    # https://goteleport.com/docs/management/diagnostics/metrics/
```

Reply:
We may also want to add a redirect for folks who already have this link copied into a local config file.